fixtures/shehd/par-zoom-18x18-rgbwa-uv.json

```
@@ -0,0 +1,160 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "PAR Zoom 18X18 RGBWA-UV",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Xavier Bongrand", "xavier bongrand"],
+    "createDate": "2024-07-09",
+    "lastModifyDate": "2024-07-09",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2024-07-09",
+      "comment": "created by Q Light Controller Plus (version 4.12.7)"
+    }
+  },
+  "physical": {
```

```suggestion
  "links": {
    "productPage": [
      "https://shehds.com/products/led-zoom-par-18x18w-rgbwauv-lighting-dj-disco-stage-lighting-good-for-party-nightclub"
    ],
    "manual": [
      "https://drive.google.com/file/d/1y-wEZ7fIC9tMded8Pu5Ykv2q3PUXeoWO/view"
    ]
  },
  "physical": {
```

Shehds has at least 3 fixtures which are 18x18 zoomable PARs -- this is the one whose channels line up with this fixture.

Then again, the specifications on the product page don't even match the user manual that it links to (different numbers of DMX channels!), so I'm not absolutely sure of anything.